src/components/CommentsList.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import { getComments } from "../../api";
+import CommentCard from "./CommentCard";
+
+function CommentsList({ article_id }) {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getComments(article_id).then((commentsData) => {
+      console.log(commentsData.data.comments);
+      setComments(commentsData.data.comments);
+    });
+  }, [article_id]);
+
```

Comment:
At articles where there are no comments yet (like No. 37), please display a message to the user to let them know.